scripts/stt_api.py

```
@@ -0,0 +1,244 @@
+
+    sliced = tmpdir / "slice.wav"
+    cmd = [ffmpeg, "-loglevel", "error", "-y"]
+    if start_s is not None:
+        cmd += ["-ss", f"{start_s:.3f}"]
+    if end_s is not None and start_s is not None:
+        cmd += ["-t", f"{(end_s - start_s):.3f}"]
+    elif end_s is not None:
+        cmd += ["-to", f"{end_s:.3f}"]
+    cmd += ["-i", str(source), "-ar", "16000", "-ac", "1", str(sliced)]
+    try:
+        subprocess.run(cmd, check=True, capture_output=True)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)